### ISSUE TITLE
All previous battlecrys trigger if you re-load hdt mid game

### ISSUE THREAD

**Issue by falconmick** [falconmick/BattleCryPluggin#2]
If you play Mysterious Challenger, end turn, close HDT, open HDT, all the custom battlecrys play.

**Comment by azeier** [comment]
Maybe this helps: `API.Core.Game.GameTime.Time` gets you the time (`DateTime`) of the current log timestamp. You could check for that to be within a certain timeframe to `DateTime.Now` before playing the sound.

**Comment by falconmick** [comment]
Yeah I planned to do somthing like that. I was just at work and decided to log the known issue so I eventually got to it. Cheers
